src/jit/lower.h

```diff
@@ -55,27 +55,63 @@ class Lowering : public Phase
             LowerNode(newNode);
         }
     }
+    void LowerRange(GenTree* firstNode, GenTree* lastNode)
+    {
+        for (GenTree* node = firstNode; node != lastNode->gtNext; node = node->gtNext)
+        {
```

Comment:
Making that constructor public is pretty dangerous: currently, it is assumed that `LIR::Range::CheckLIR` will return `true` for any `Range`. Violating that assumption may e.g. cause `TryGetUse` to return incorrect results if the user of the node is not in the created range.